packages/arb-token-bridge-ui/tests/e2e/specs/withdrawERC20.cy.ts

```diff
@@ -109,6 +124,68 @@ describe('Withdraw ERC20 Token', () => {
       })
     })
 
+    it('should claim funds', { defaultCommandTimeout: 200_000 }, () => {
+      // increase the timeout for this test as claim button can take ~(20 blocks *10 blocks/sec) to activate
+
+      cy.login({ networkType: 'parentChain' }) // login to L1 to claim the funds (otherwise would need to change network after clicking on claim)
+
+      cy.findByLabelText('Open Transaction History')
+        .should('be.visible')
+        .click()
+
+      cy.waitUntil(
+        () =>
+          cy
+            .findByLabelText(
+              `Claim ${formatAmount(ERC20AmountToSend, {
+                symbol: 'WETH'
+              })}`
+            )
+            .should('be.visible')
+            .should('not.be.disabled'),
+        {
+          errorMsg: 'Claim Transaction button is not visible or enabled',
+          timeout: 200_000,
+          interval: 1000
+        }
+      ).then(() => {
+        cy.findByLabelText(
+          `Claim ${formatAmount(ERC20AmountToSend, {
+            symbol: 'WETH'
+          })}`
+        )
+          .click()
+          .then(() => {
+            cy.confirmMetamaskTransaction().then(() => {
+              cy.findByLabelText('show settled transactions')
+                .should('be.visible')
+                .click()
+
+              cy.findAllByText(
+                `${formatAmount(ERC20AmountToSend, {
+                  symbol: 'WETH'
+                })}`
+              )
+                .first()
+                .should('be.visible')
+
+              cy.findByLabelText('Close side panel').click()
+
+              cy.searchAndSelectToken({
+                tokenName: 'WETH',
+                tokenAddress: wethTokenAddressL2
+              })
+
+              // the balance on the destination chain should not be the same as before
+              cy.findByLabelText('WETH balance amount on parentChain')
```

Comment:
nit: we could probably assert for should equal `l1ERC20Bal + ERC20AmountToSend`

Reply:
Ideally, yes - but this would involve updating the existing code because `l1ERC20Bal` is type string (value obtained after doing `formatAmount(balance, 'ETH')`, and `ERC20AmountToSend` is pure number. Adding them is not straightforward so I would like to park it for a general cleanup later.